README.md

```diff
@@ -1,49 +1,97 @@
+- **Modernizing the Codebase**: Transitioning to modern C++ standards and refactoring old code.
+- **Critical Bug Fixes**: Fixing game-breaking issues (e.g., fullscreen crash).
+- **Minor Bug Fixes**: Addressing minor bugs (e.g., UI issues, graphical glitches).
+- **Cross-Platform Support**: Adding support for more platforms (e.g., Linux, macOS).
+- **Engine Improvements**: Enhancing the game engine to improve performance and stability.
+- **Client-Side Features**: Enhancing the game’s client with features such as an improved replay viewer and UI updates.
+- **New Gameplay Features**: Adding new in-game features (e.g., new units, balance changes).
+- **Multiplayer Improvements**: Implementing a new game server and an upgraded matchmaking lobby.
+- **Tooling Improvements**: Developing new or improving existing tools for modding and game development.
+- **Community-Driven Improvements**: Once the community grows, we plan to incorporate more features, updates, and
+  changes based on player feedback.
 
-The quickest way to build all configurations in the project is to open `rts.dsw` in Microsoft Visual Studio C++ 6.0 (SP6 recommended for binary matching to Generals patch 1.08 and Zero Hour patch 1.04) and select Build -> Batch Build, then hit the “Rebuild All” button.
+## Running the Game
 
-If you wish to compile the code under a modern version of Microsoft Visual Studio, you can convert the legacy project file to a modern MSVC solution by opening `rts.dsw` in Microsoft Visual Studio .NET 2003, and then opening the newly created project and solution file in MSVC 2015 or newer.
+To run *Zero Hour* using this project, you need to have the original *Command & Conquer: Generals and Zero Hour* game
+installed. The easiest way to get it is through *Command & Conquer The Ultimate Collection*
+on [Steam](https://store.steampowered.com/bundle/39394). Once the game is ready, download the latest version of the
+project from [GitHub Releases](https://github.com/TheSuperHackers/GeneralsGameCode/releases), extract the necessary 
+files, and follow the instructions in the [Wiki](https://github.com/TheSuperHackers/GeneralsGameCode/wiki).
 
-NOTE: As modern versions of MSVC enforce newer revisions of the C++ standard, you will need to make extensive changes to the codebase before it successfully compiles, even more so if you plan on compiling for the Win64 platform.
+### Disclaimer
 
-When the workspace has finished building, the compiled binaries will be copied to the folder called `/Run/` found in the root of each games directory. 
+The current builds may have bugs or unexpected behavior. We will aim to release stable versions that are as free from
+bugs as possible, but even those may have some unexpected behaviors.
 
+### Antivirus Warning
 
-## Known Issues
+Some antivirus software may mistakenly flag this project as a false positive (e.g., Windows Defender). We assure you
```

Comment:
Is this true? Have we seen that already? This again may put the reader at unease.